craft_application/commands/lifecycle.py

```diff
@@ -77,32 +77,27 @@ def _fill_parser(self, parser: argparse.ArgumentParser) -> None:
             help="Build in a LXD container.",
         )
 
-    @override
-    def get_managed_cmd(self, parsed_args: argparse.Namespace) -> list[str]:
-        cmd = super().get_managed_cmd(parsed_args)
-
-        cmd.extend(parsed_args.parts)
-
-        return cmd
-
     @override
     def provider_name(self, parsed_args: argparse.Namespace) -> str | None:
         return "lxd" if parsed_args.use_lxd else None
 
-    @override
-    def run_managed(self, parsed_args: argparse.Namespace) -> bool:
-        """Return whether the command should run in managed mode or not.
+    def _run_manager_for_build_plan(self, fetch_service_policy: str | None) -> None:
```

Comment:
```suggestion
    def _run_managed_for_build_plan(self, fetch_service_policy: str | None) -> None:
```

possible typo?

Reply:
Intentional, but worth discussing. The intent is that this method would be what actually manages the instances for the build plan, so it would be the "manager".